.github/workflows/rust.yaml

```diff
@@ -101,11 +101,19 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v2
+      - run: echo "VCPKG_ROOT=$env:VCPKG_INSTALLATION_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
```

Comment:
Could you try add this after the current line?

```
      # https://github.com/sfackler/rust-openssl/issues/1062#issuecomment-489441940
      - run: echo "VCPKGRS_DYNAMIC=1" | Out-File -FilePath $env:GITHUB_ENV -Append
```

If it still not work, we may try https://github.com/sfackler/rust-openssl/issues/1062#issuecomment-555202055

Reply:
Added. Let's see 🤞